src/sentry/templates/sentry/integrations/slack-link-team.html

```diff
@@ -18,15 +18,15 @@
     <p>Choose which team you'd like to link to the <b>#{{ channel_name }}</b> channel.</p>
 
     <div class="align-center" style="padding-bottom:20px;">
-      <select id="{{ form.team.id_for_label }}" name="{{ form.team.html_name }}" class="form-control">
+      <select id="{{ form.team.id_for_label }}" data-test-id="select-team" name="{{ form.team.html_name }}" class="form-control">
```

Comment:
You could also query by the `name` attribute e.g. `[name="<whatever form.team.html_name is>"]`